scripts/install_narrative.sh

```diff
@@ -124,6 +124,8 @@ then
         exit 1
     fi
 
+    # Install sklearn and clustergrammer
```

Comment:
added some extra text to make it easier to compare with the `install_narrative_docker.sh` script (below)